test/includes/microcloud.sh

```diff
@@ -629,6 +628,12 @@ validate_system_lxd() {
     fi
 
     lxc remote switch local
+
+    # Remove the trust on the remote which was added when adding the remote.
+    fingerprint="$(lxc query microcloud-test:/1.0/certificates?recursion=1 | jq -r '.[] | select(.name=="test") | .fingerprint')"
+    lxc config trust remove "microcloud-test:${fingerprint}"
```

Comment:
Thanks, I think this should be no longer required - after backports to 5.21. Similarly, lxc 6.1 should now work with 5.21 (if still required).

Reply:
I was thinking it would still be good to keep it in for clean up purposes? Meaning if you add the remote again, it also reads the trust on the remote.